.github/workflows/rust.yml

```diff
@@ -47,6 +49,24 @@ jobs:
         uses: Swatinem/rust-cache@ce325b60658c1b38465c06cc965b79baf32c1e72
       - name: cargo build
         run: cargo build --features=${{ env['CARGO_PROJECT_FEATURES'] }}
+  build-git-libostree:
```

Comment:
Minor/optional: would be nice to give it a name following a similar convention as the other ones.